### PR TITLE
Auto Scaling Group: c03-aws01

### PR DESCRIPTION
**Pull request by Faria-Marcio**
# Description

Exercise (e.g. C01-E01): c03-aws01

Student Name: Marcio Faria

>AWS - Auto Scaling Group

## Type of change

- [ X ] Exercise Submission (one exercise per PR)
- [ X ] This PR is just for **one** exercise of a class
- [ X ] PR name follows the pattern `<github-username>/<exercise-number>`
- [ X ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
- [ X ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/03class/exercises/c03-aws01/Faria-Marcio/c03-aws01.txt [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	- http://52.63.182.63:80
+	    ip-192-168-101-151.ap-southeast-2.compute.internal
+
+
+   - Details of the security group used
+	Name	Group ID		Group Name		VPC ID			Type	Description			Owner
+		sg-0f0a8b39374d935a7	devopsacademy-sg-asg	vpc-04386a3405acb18af	EC2-VPC	AutoScaling-Security-Group	148446344264
+
+
+   - What this line is doing? curl https://169.254.169.254/latest/meta-data/hostname > index.html
```
Comment (caiocezart) [inline classes/03class/exercises/c03-aws01/Faria-Marcio/c03-aws01.txt]: can you add more information about the 169.254.169.254 url?

Comment (Faria-Marcio) [reply classes/03class/exercises/c03-aws01/Faria-Marcio/c03-aws01.txt]: 169.254.169.254 is used in Amazon EC2 and other cloud computing platforms to distribute metadata to cloud instances.

References:
   https://serverfault.com/questions/427018/what-is-this-ip-address-169-254-169-254/427022
   https://stackoverflow.com/questions/42314029/whats-special-about-169-254-169-254-ip-address-for-aws

Comment (caiocezart) [reply classes/03class/exercises/c03-aws01/Faria-Marcio/c03-aws01.txt]: right, can you add that to your answer please?

Comment (Faria-Marcio) [reply classes/03class/exercises/c03-aws01/Faria-Marcio/c03-aws01.txt]: Omg I forgot to update the file. 
Done

**Pull request merged**